packages/renderer/src/ffmpeg-flags.ts

```diff
@@ -1,19 +1,29 @@
 import execa from 'execa';
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import {_downloadFile} from './browser/BrowserFetcher';
+import type {FfmpegExecutable} from './ffmpeg-executable';
 import {binaryExists} from './validate-ffmpeg';
 
 let buildConfig: string | null = null;
+const listeners: Record<string, ((path: string) => void)[]> = {};
+const isDownloading: Record<string, boolean> = {};
 
 export type FfmpegVersion = [number, number, number] | null;
 
-export const getFfmpegBuildInfo = async (options: {
-	ffmpegExecutable: string | null;
-}) => {
+export const getFfmpegBuildInfo = async (
+	options: {
+		ffmpegExecutable: string | null;
+	},
```

Comment:
Better to either make an object with options, or multiple arguments rather than mixing them

Reply:
(I realize now i did this myself 😂)